src/polar/Pie.tsx

```diff
@@ -305,17 +307,15 @@ function PieSectors(props: PieSectorsProps) {
   });
 }
 
-const getRealPieData = (item: Pie): RealPieData[] => {
-  const { data, children } = item.props;
+const getRealPieData = (item: Pie, cells: ReactElement[]): RealPieData[] => {
```

Comment:
I was in horror when I discovered that Recharts likes to randomly mutate arrays here and there so I am trying to switch everything to `ReadonlyArray` since to help me find those places. Is it a big change to do that here too?

Reply:
nope - done